### ISSUE TITLE
Most prominent Windows download does not register to handle bitcoincash: URIs

### ISSUE THREAD

**Issue by mhsmith** [Electron-Cash/Electron-Cash#1504]
On https://electroncash.org/, the first option in the list is the stand-alone EXE, so that's probably what most people download. But that doesn't register with the system to handle bitcoincash: URIs.

**Comment by cculianu** [comment]
I agree with you.

@fyookball  -- you control the website. Want to consider this issue?

**Comment by fyookball** [comment]
@mhsmith That's an interesting point.  I'll consider it.

Historically, the standalone executable has always been the main offering both on electrum and electron cash.  My personal opinion is that system installs on windows are less tasteful and less with the "crypto ethos" than a standalone.

**Comment by cculianu** [comment]
@fyookball I understand the history but I want to point out that don't we want to be friends with BitPay now? Installing has the advantage that on Windows, clicking a `bitcoincash:` URL actually then works and opens up Electron Cash (because the install process registers the `bitcoincash:` URL with Windows).

Please reconsider this.  We want the ecosystem to function smoothly.  We also want to be friends with BitPay.  

Thanks,

-Calin

**Comment by EchterAgo** [comment]
I also much prefer the standalone over the install. I could make the standalone have the option to register the URL handler.

**Comment by cculianu** [comment]
Oh wow - this would be the best of all worlds.  By writing to the registry I guess?

Now the dilemma is do you do it automatically (my personal preference would be YES), or do you have it in a tools options buried somewhere...

**Comment by mhsmith** [comment]
[Here](https://www.reddit.com/r/btc/comments/d80cil/on_electron_cash_409_android_bip70_uris_only_work/) is a person who was so accustomed to manually copying URIs from their browser and pasting them into Electron Cash, that when they couldn't do the same on Android, they went and wrote a detailed 500 word description of the problem on Reddit before it ever occurred to them to simply click the link in the browser.

**Comment by cculianu** [comment]
Ha ha ha -- classic internet.

**Comment by mhsmith** [comment]
Yeah, but I didn't post this to mock them, but to suggest that this issue be reopened. We shouldn't be forcing users to learn workarounds like this.

**Comment by cculianu** [comment]
Good point.

@EchterAgo I know you're busy with other stuff but we shall remain eternally grateful if you do the winapi calls in Python to register the exe some day! Reopening now...
